test/lint/lint_ignore_dirs.py

```diff
@@ -0,0 +1,5 @@
+EXCLUDED_DIRS = ["src/leveldb/",
```

Comment:
Please name it `SHARED_EXCLUDED_DIRS`, that is clearer and I don't like that the same var name is reused in the whitespace and includes linters (also means it's harder to grep for).

Reply:
renamed `EXCLUDED_DIRS` to `SHARED_EXCLUDED_DIRS`